### ISSUE TITLE
Separate configuration options for showing footnotes and cross-references

### ISSUE THREAD

**Issue by benasocj** [AndBible/and-bible#2496]
Upon opening the settings menu for a workspace, it's currently only possible to show or hide *both*, footnotes *and* cross-references. It would be great if these could be separate configuration options since I usually want to display footnotes but hide cross-references for better readability.

**Comment by tuomas2** [comment]
Makes sense to me. Would you be interested to implement this?

**Comment by benasocj** [comment]
Sorry, but I don't think I have the experience nor the time to do so.

**Comment by agrogers** [comment]
@tuomas2 , I have added this to the Options but I don't understand why the VUE file doesn't do what I think it should.

`app/bibleview-js/src/components/OSIS/Note.vue`

First, config.showFootNotes and my new config.showXrefs works fine - I have verified that.

Second, the OR in this code seems redundant to me.
```
const showHandle = computed(() => {
    return !exportMode.value && ((config.showFootNotes && isCrossReference) || config.showFootNotes);
});
```

Third, this was my code which I reckon should work. It still shows both footnotes and cross references.
```
const showHandle = computed(() => {
    return !exportMode.value && ((config.showFootNotes && isFootNote) || (config.showXrefs && isCrossReference));
});
```

Finally, if I replace the 'return' with the following, BOTH footnotes and cross references still show.

`return !exportMode.value && isCrossReference && isFootNote`

But the css Class config works fine - it includes the correct class name based on these variables.

```
  <span
    v-if="showHandle"
    class="skip-offset">
    <span class="highlight-transition" :class="{isHighlighted: showNote, noteHandle: true, isFootNote, isCrossReference, isOther}" @click="noteClicked">
      {{handle}}
    </span>
  </span>
```

Can you explain what is wrong here? It seems the isFootNote and isCrossReference work in the formatting code but not in the script. In the script, they are both true regardless of what type of note is being displayed.

**Comment by tuomas2** [comment]
isCrossReference is a computed reference so we should have 
```
const showHandle = computed(() => {
    return !exportMode.value && ((config.showFootNotes && isCrossReference.value) || config.showFootNotes);
});
```
fixed in  887735362930719f3a5e46c949c3228137819c54

In your case
```
const showHandle = computed(() => {
    return !exportMode.value && ((config.showFootNotes && isFootNote.value) || (config.showXrefs && isCrossReference.value));
});
```

**Comment by agrogers** [comment]
PR #2503 should address this.
